packages/user_packages/iotcc_mqtt_device.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+# ----------------------------------------------------------------------------#
+#  Copyright © 2015-2016 VMware, Inc. All Rights Reserved.                    #
```

Comment:
I agree with Swarnadeep that generally in the copyright, current year is mentioned on creation of a new file. Up to you if you want to change it.

Reply:
We have been using this license comment at top of every Liota code file and if we change here it will require changes in the entire liota codebase, probably it should be reviewed from the OSL team before making this change, I`ll prefer to keep it same for now.

Reply:
ok, no issues.